frontend/App.tsx

```diff
@@ -57,7 +55,7 @@ export default function App() {
   }
 
   useEffect(() => {
-    AsyncStorage.getItem(HAS_SEEN_ONBOARDING).then(val => setStartingState(val ? "SignInFlow" : "Onboarding"));
+    setStartingState("SignInFlow");
```

Comment:
I don't think we need startingState anymore - double check me, but if that's true then we can just remove it.

Reply:
removed - initial route is now set to "SignInFlow" for navigation